src/Operations.jl

```diff
@@ -743,6 +743,28 @@ function download_source(ctx::Context, pkgs::Vector{PackageSpec},
     return new_pkgs
 end
 
+function try_prompt_pkg_add(mod::Symbol)
+    ctx = Context()
+    pkg = PackageSpec(String(mod))
+    registry_resolve!(ctx.registries, pkg)
+    !Types.isresolved(pkg) && return false
+    resp = try
+        Base.prompt(" \033[32m\033[1m└\033[22m\033[0m Package $(repr(String(mod))) not found, but $(repr(String(mod))) is available from a registry.
+   Active project: $(Base.active_project())
+   Add $(repr(String(mod))) to the active project? (y/n)", default = "n")
+    catch err
+        if err isa InterruptException
+            println()
+            return false
+        end
```

Comment:
Were you intending to swallow all exceptions silently here?

```suggestion
        end
        rethrow()
```

Reply:
Nope, thanks!